assets/js/nav.js

```diff
@@ -0,0 +1,17 @@
+const navChevron = document.querySelectorAll(".chevron-down");
+const chevronUp = "/theme-assets/chevron-up.svg";
+const chevronDown = "/theme-assets/chevron-down.svg";
+
+for (let i = 0; i < navChevron.length; i++) {
```

Comment:
since the index itself isn't required ('cause we're not doing any `foo[i+1]`-style neighbor searching), I would simplify this to

```js
for (const element of document.querySelectorAll('...')) {
```
Since (imo):

* It saves an extra const declaration that won't be used later anyway
* makes it clear that you're not going to replace any of the elements you find
* makes the code a bit more readable than `navChevron[i]` (you could replace `element` with `navElement` if that makes it more readable though)